src/IOCapture.jl

```diff
@@ -98,7 +102,8 @@ function capture(
     rethrow::Type=Any,
     color::Bool=false,
     passthrough::Bool=false,
-    capture_buffer=IOBuffer()
+    capture_buffer=IOBuffer(),
+    io_context::Vector=[],
```

Comment:
Should it be `AbstractVector`?

```suggestion
    io_context::AbstractVector=[],
```